### PR TITLE
OSSM-4463: Link out to KCS from relevant docs location

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4463

Link to docs preview:

Link to ROSA preview build that contains the link to the KCS article: https://62953--docspreview.netlify.app/openshift-rosa/latest/service_mesh/v2x/ossm-create-smcp.html

Link to OCP Enterprise doc preview build that does not contain the link to the KCS article: https://62953--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-smcp.html

QE review:
- [x] QE has approved this change.

Additional information:

Aligned team: Doc Devtools

Dev Review: mayleighnmyers
QA Review: pbajjuri20
Peer Review: bburt-rh

### CODE REVIEW

**Comment by mayleighnmyers** [comment]
LGTM

**Comment by pbajjuri20** [comment]
LGTM...!

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-about-smcp.adoc [hunk 1]
```diff
@@ -21,3 +21,7 @@ Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where
 ====
 The {SMProductShortName} documentation uses `istio-system` as the example project, but you can deploy the service mesh to any project.
 ====
+
+ifndef::openshift-enterprise[]
+If you are deploying the control plane for use on Red Hat OpenShift on AWS (ROSA), see the https://access.redhat.com/solutions/6529231[KCS article] that discusses adding a new project and starting pods.
```
Comment (bburt-rh) [inline modules/ossm-about-smcp.adoc:26]:
```suggestion
If you are deploying the control plane for use on Red Hat OpenShift on AWS (ROSA), see the link:https://access.redhat.com/solutions/6529231[KCS article] that discusses adding a new project and starting pods.
```
- Should `Red Hat OpenShift on AWS (ROSA)` use an attribute?
- hyperlink should use `link:` at the beginning
- Per RHSSG, "Use the title of the Knowledgebase article for the link text, or use descriptive running text" : https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links

**Comment by bburt-rh** [comment]
Don't forget to remove `WIP` from the title.

File: modules/ossm-about-smcp.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-about-smcp.adoc]: Changes were made. 

I created a rosa attribute in common-attributes and added it to the service mesh section.

**Comment by rh-tokeefe** [comment]
Rebased PR ~ 3:15 p.m. on 8/4/23

File: modules/ossm-about-smcp.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/ossm-about-smcp.adoc]: @rh-tokeefe If this change is for ROSA only, why not use `ifdef::openshift-rosa`?

**Comment by rh-tokeefe** [comment]
@mburke5678 as a member of the aligned Dev Tools doc teams, and not the core OpenShift doc team, I'm new to this process. Is there anything else I need to do at this time to have the PR formally reviewed and merged?

File: modules/ossm-about-smcp.adoc [hunk 2]
```diff
@@ -21,3 +21,7 @@ Red Hat OpenShift Service on AWS (ROSA) places additional restrictions on where
 ====
 The {SMProductShortName} documentation uses `istio-system` as the example project, but you can deploy the service mesh to any project.
 ====
+
+ifndef::openshift-enterprise[]
+If you are deploying the control plane for use on {product-rosa}, see the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6529231[OpenShift service mesh operator istio basic not starting due to authentication errors], which discusses adding a new project and starting pods.
```
Comment (bburt-rh) [inline modules/ossm-about-smcp.adoc:26]:
```suggestion
If you are deploying the control plane for use on {product-rosa}, see the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6529231[OpenShift service mesh operator Istio basic not starting due to authentication errors], which discusses adding a new project and starting pods.
```
- Should Istio be capitalized? (even though it is not in the actual article title)

**Pull request merged**